core/src/main/scala/kafka/utils/Json.scala

```diff
@@ -69,8 +70,11 @@ object Json {
    * @return An `Either` which in case of `Left` means an exception and `Right` is the actual return value.
    */
   def tryParseFull(input: String): Either[JsonProcessingException, JsonValue] =
-    try Right(mapper.readTree(input)).map(JsonValue(_))
-    catch { case e: JsonProcessingException => Left(e) }
+    if (input != null && input.isEmpty)
```

Comment:
Wasn't sure what to do here but `null` is checked in `readTree` and an IllegalArgumentException is thrown so I decided to go skip here and leave the original behavior.

Reply:
I think we should say `if (input == null || input.isEmpty)` instead. It's unexpected to get an exception instead of `Left` in a method that returns `Either`.

Reply:
Any update regarding this?

Reply:
Made the required changes, updated the related test too.